src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyIndexPage.tsx

```diff
@@ -80,7 +80,7 @@ const MethodologyIndexPage: NextPage<Props> = ({ themes = [] }) => {
                 <ul className="govuk-!-margin-top-0">
                   {orderBy(getMethodologiesForTopics(topics), 'title').map(
                     methodology => (
-                      <li key={methodology.id}>
+                      <li key={methodology.id} id="methodology-heading">
```

Comment:
Similar thing to other comment, ids should be unique. `methodology-heading` is also a weird name as well.

This really doesn't need any kind of id at all, we can just use a selector on `li` within the accordion section